navpy/core/tests/test_navpy.py

```diff
@@ -117,6 +117,25 @@ def test_ned2ecef(self):
         
         for e1, e2 in zip(ecef_computed, ecef):
             self.assertAlmostEqual(e1, e2, places=3)
+            
+    
+    def test_ecef2lla_equator(self):
+        """
+        Test conversion of ECEF to LLA from the equator
+        
+        LLA = [0, 0 ,0], ECEF = [6378137. , 0., 0.]
+        """
+        lat = 0.; lon = 0.; alt = 0.;
+        ecef = np.array([6378137. , 0., 0.])
```

Comment:
Would it make sense to reference the equatorial earth radius parameter as opposed to hard coding the value?  Below is a suggestion.
```suggestion
        ecef = np.array([navpy.wgs84.a, 0., 0.])
```